docs/source/python36.rst

```diff
@@ -1,67 +0,0 @@
-.. _python-36:
```

Comment:
Why was this deleted?

Reply:
I figured a "what's new in Python 3.6" page doesn't make sense when Python 3.6 is the oldest supported Python and we don't have similar pages for any other Python release. Everything on this page is covered in another section already.